HLTrigger/HLTcore/plugins/TriggerSummaryAnalyzerRAW.cc

```diff
@@ -157,6 +157,35 @@ void TriggerSummaryAnalyzerRAW::analyze(edm::StreamID, const edm::Event& iEvent,
       if (nL1TEtSum > 0)
         LogVerbatim("TriggerSummaryAnalyzerRAW") << " L1TEtSum: " << nL1TEtSum;
 
+      /* Phase-2 */
+      const unsigned int nL1TTkMuon(handle->l1ttkmuonSlice(iFO).second - handle->l1ttkmuonSlice(iFO).first);
+      if (nL1TTkMuon > 0)
+        LogVerbatim("TriggerSummaryAnalyzerRAW") << " L1TTkMuon: " << nL1TTkMuon;
+
+      const unsigned int nL1TTkEle(handle->l1ttkeleSlice(iFO).second - handle->l1ttkeleSlice(iFO).first);
+      if (nL1TTkEle > 0)
+        LogVerbatim("TriggerSummaryAnalyzerRAW") << " L1TTkEle: " << nL1TTkEle;
+
+      const unsigned int nL1TTkEm(handle->l1ttkemSlice(iFO).second - handle->l1ttkemSlice(iFO).first);
+      if (nL1TTkEm > 0)
+        LogVerbatim("TriggerSummaryAnalyzerRAW") << " L1TTkEm: " << nL1TTkEm;
+
+      const unsigned int nL1TPFJet(handle->l1tpfjetSlice(iFO).second - handle->l1tpfjetSlice(iFO).first);
+      if (nL1TPFJet > 0)
+        LogVerbatim("TriggerSummaryAnalyzerRAW") << " L1TPFJet: " << nL1TPFJet;
+
+      const unsigned int nL1TPFTau(handle->l1tpftauSlice(iFO).second - handle->l1tpftauSlice(iFO).first);
+      if (nL1TPFTau > 0)
+        LogVerbatim("TriggerSummaryAnalyzerRAW") << " L1TPFTau: " << nL1TPFTau;
+
+      const unsigned int nL1THPSPFTau(handle->l1tetsumSlice(iFO).second - handle->l1tetsumSlice(iFO).first);
+      if (nL1THPSPFTau > 0)
+        LogVerbatim("TriggerSummaryAnalyzerRAW") << " L1THPSPFTau: " << nL1THPSPFTau;
+
+      const unsigned int nL1TPFTrack(handle->l1tetsumSlice(iFO).second - handle->l1tetsumSlice(iFO).first);
```

Comment:
```suggestion
      const unsigned int nL1TPFTrack(handle->l1tpftrackSize(iFO).second - handle->l1tpftrackSize(iFO).first);
```

Reply:
All addressed, but it was "Slice", not "Size" -- that's why I didn't accept the Suggestion here. Thanks for the review!